.github/workflows/score_history.yml

```diff
@@ -69,17 +72,37 @@ jobs:
       - name: Install dependencies
         run: pip install numpy pandas requests python-dotenv GitPython sqlalchemy psycopg2
 
+
+      - name: Add scalingo to know hosts
+        run: echo "KNOWN_HOSTS=$(ssh-keyscan -H ssh.osc-fr1.scalingo.com)" >> $GITHUB_ENV
+
+      - name: Install SSH key
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.SCALINGO_SSH_KEY }}
```

Comment:
I'm cursious as to where does this `SCALINGO_SSH_KEY` value come from?

Reply:
This is a private key generated with `ssh-keygen -t ed25519`. I've added the corresponding public key to my scalingo account. we need to do the same with a scalingo dedicated user. Discussion about it is currently ongoing here: https://mattermost.incubateur.net/betagouv/pl/mdpjyet487nqmdw8iw8gzdmrfc